wp-content/themes/workingnyc/src/scss/utilities/margins/_margins.scss

```diff
@@ -0,0 +1,26 @@
+/**
+ * Margins
+ */
+
+// Creating this class as a copy of the tailwind class because the import isn't working
+.mx-28 {
+    margin-left: 8rem; /* 128px */
+    margin-right: 8rem; /* 128px */
+  }
+.mx-50 {
+    margin-left: 200px;
+    margin-right: 200px;
+  }
+.mr-12 {
+    margin-right: 48px;
+  }
+  .custom-padding {
```

Comment:
custom-padding, custom-margin, and custom-margin-display are not descriptive names, and would be confusing for another engineer. Can you change them to tailwindcss names so that we can replace them with tailwind in the future? For example, custom-padding would be `pl-10`. You may want to split custom-margin-display into two classes.

Reply:
<div class="mobile-py-33 tablet:py-3 desktop:py-3 desktop-mx-50 tablet-mx-14">
I have used these custom classes using the media query css and it's working locally